rdt-open-reader/src/covidIndonesia/assets/json.form/patient-registration-form-original.json

```diff
@@ -186,7 +186,7 @@
         "hint": "Date of birth",
         "expanded": false,
         "duration": {
-          "label": "DoB"
+          "label": "Age"
```

Comment:
I think this is a dob and not age field.

Reply:
yes, reverted

Reply:
Can you just confirm for me the first issue of file:///home/vend/Downloads/Translation.issues_SID.pdf this PDF which is related the to DOB field so In thePDF says "Tanggal lahir" to "Usia" so for this reason I have changed the "DoB" to "Age" so could you suggest me the correct fix of this issue

Reply:
The change in Bahasa shouldn't affect the English version.